backends/bmv2/simple_switch/simpleSwitch.cpp

```diff
@@ -782,6 +782,10 @@ SimpleSwitchBackend::convertChecksum(const IR::BlockStatement *block, Util::Json
                     cksum->emplace("target", jleft->to<Util::JsonObject>()->get("value"));
                     cksum->emplace("type", "generic");
                     cksum->emplace("calculation", calcName);
+                    if (verify)
+                        cksum->emplace("verify", true);
+                    else
+                        cksum->emplace("update", true);
```

Comment:
Antonin said in a comment that BMv2 defaults to the value 'true' on these two keys, if they are absent.  I think it would be best in both cases to add both keys, both with explicit true/false values, otherwise I doubt it will fix the problem.